src/utils/index.ts

```diff
@@ -35,6 +36,26 @@ export const getGitOrgData = async (what: WhatEnum, org: string, page = 1): Prom
 
 // ************************************************************ //
 
+export const setTeamsData = async ( ): Promise<void> => {
+    try {
+        for (const team of ORG_DATA.teams) {
+            const members: string = team["members_url"];
```

Comment:
Suggestion, alternative method to extract the members url. `"members_url"` property value itself is difficult as it has `{/members}` at the end, which forces usage of `.slice()` with hard-coded value `9` to extract url. Instead, we can construct members url from the base `"url"` property.

```
const membersUrl: string = `${team["url"]}/members`;
const membersData = await fetch(membersUrl, HEADERS);
```

Thoughts on this approach?

Reply:
I like it, it is more clean, the variable name has to be changed as well, instead of `membersUrl` should be `teamUrl`.